src/server/dflycmd.h

```diff
@@ -214,4 +206,6 @@ class DflyCmd {
   Mutex mu_;  // Guard global operations. See header top for locking levels.
 };
 
+std::string_view SyncStateName(DflyCmd::SyncState sync_state);
```

Comment:
Wouldn't it be more elegant to place this function within `DflyCmd`, next to the definition for `SyncState` itself?